.circleci/config.yml

```diff
@@ -300,15 +313,15 @@ jobs:
       - run: yarn grunt prepareDeploy
         # Install dependencies for Firebase functions to prevent parsing errors during deployment
         # See https://github.com/angular/angular.js/pull/16453
-      - run: yarn -cwd $PROJECT_ROOT/scripts/docs.angularjs.org-firebase/functions
+      - run: yarn -cwd ~/ng/scripts/docs.angularjs.org-firebase/functions
       - run: yarn firebase deploy --token "$FIREBASE_TOKEN" --only hosting
 
   deploy-code:
     executor:
       name: cloud-sdk
     steps:
       - custom_attach_workspace
-      - init_environment
+      - run: ls ~/ng/deploy/code
```

Comment:
Would it work to set `working_directory` to `~/ng` for the `cloud-sdk` executor and adjust paths accordingly?

Reply:
I'll try it.

Reply:
It didn't work... perhaps because the `~/ng` folder doesn't exist until the workspace is attached??  Anyway, it works like this so I am going to leave it as-is.